### PR TITLE
LPS-90943 Add data collection search

### PR DESCRIPTION
**Pull request by mclomello**
@gabrielwas

### CODE REVIEW

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/service/DEDataRecordCollectionSearchRequest.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.data.engine.service;
+
+import com.liferay.portal.kernel.dao.orm.QueryUtil;
+
+/**
+ * This class represents a request to get a list of {@link DEDataRecordCollection}
```
[the remark this answers is not in the document]
Comment (mclomello) [reply modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/service/DEDataRecordCollectionSearchRequest.java]: Sure Gabriel, I'll fix this.

File: modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/service/DEDataRecordCollectionSearchRequest.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
+/**
```
[the remark this answers is not in the document]
Comment (mclomello) [reply modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/service/DEDataRecordCollectionSearchRequest.java]: Also fix that to.

File: modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java [hunk 3]
```diff
@@ -577,6 +598,153 @@ protected DEDataEnginePermissionSupport getDEDataEnginePermissionSupport() {
 		return new DEDataEnginePermissionSupport(groupLocalService);
 	}
 
+	protected DEDataEngineRequestExecutor getDEDataEngineRequestExecutor() {
```
[the remark this answers is not in the document]
Comment (mclomello) [reply modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java]: Yes, I'll fix that to.

**Comment by mclomello** [comment]
ci:test:sf

**Comment by mclomello** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
Pull request submitted to https://github.com/gabrielwas/liferay-portal/pull/31. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**